externs/google_visualization_api.js

```diff
@@ -0,0 +1,480 @@
+// Copyright 2009 Google Inc.
+// All Rights Reserved.
```

Comment:
Yes, but I ended up just using the publicly available outdated gviz externs and added missing methods.